reference/orders.v2.oas2.yml

```
@@ -693,7 +693,7 @@ paths:
 
         Acceptable values for `shipping_provider` include an empty string (`""`), auspost, canadapost, endicia, usps, fedex, royalmail, ups, upsready, upsonline, or shipperhq.
 
-        Acceptable values for `tracking_carrier` include an empty string (`""`) or one of the valid tracking-carrier values viewable [here](https://github.com/bigcommerce/dev-docs/blob/master/assets/csv/tracking_carrier_values.csv) and downloadable as a .CSV file [here](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/csv/tracking_carrier_values.csv).
+        Acceptable values for `tracking_carrier` include an empty string (`""`) or one of the valid [tracking-carrier values](https://github.com/bigcommerce/dev-docs/blob/master/assets/csv/tracking_carrier_values.csv) and the [downloadable .CSV file](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/csv/tracking_carrier_values.csv).
```

```suggestion
        Acceptable values for `tracking_carrier` include an empty string (`""`) or one of the valid [tracking-carrier values](https://github.com/bigcommerce/dev-docs/blob/master/assets/csv/tracking_carrier_values.csv).
```
@slsriehl This is my suggested edit. I believe you're saying that we should only have one link? I see that people can get the raw version (second link) from the first link (as long as they can use github), so I took off the second link. 

I'll change the rest of them if it works.